compiler/src/Language/Bond/Codegen/Cpp/Types_h.hs

```diff
@@ -118,8 +119,8 @@ namespace std
     {
         #{newlineSepEnd 2 field structFields}#{defaultCtor}
 
-        #{copyCtor}
-        #{moveCtor}
+        #{copyCtor}#{optional allocatorCopyCtor allocator}
+        #{moveCtor}#{optional allocatorMoveCtor allocator}
```

Comment:
Consider moving the `optional` into `allocatorCopyOrMoveCtor` so we do the two checks at the same place.

Reply:
Updated.